readme.md

```diff
@@ -18,41 +18,67 @@ Enable Accelerated Mobile Pages (AMP) on your WordPress site.
 
 Bring the speed and features of the open source [AMP project](https://www.ampproject.org/) to your site, the WordPress way.
 
-With the plugin active, all posts on your site will have AMP-compatible versions, accessible by appending `/amp/` to the end your post URLs. For example, if your post URL is `http://example.com/2016/01/01/amp-on/`, you can access the AMP version at `http://example.com/2016/01/01/amp-on/amp/`. If you do not have [pretty permalinks](https://codex.wordpress.org/Using_Permalinks#mod_rewrite:_.22Pretty_Permalinks.22) enabled, you can do the same thing by appending `?amp=1`, i.e. `http://example.com/?p=123&amp=1`
+Run AMP on your entire site, with your own theme. This can mean little or no custom development to get the same experience in AMP.
```

Comment:
I think that “This can mean little or no custom development to get the same experience in AMP” could make it sound a bit too easy to start with. In reality there will often be a good amount of dev work that is needed when activating on a non-core theme on an existing site.

Reply:
Good point, I'll change this.

Reply:
How does this look? https://github.com/Automattic/amp-wp/pull/1561/commits/efec95c214172c00f659f26f7b43c32c48865cf2#diff-eb6b6c90251ab33cee784713c451e6d8L17 

I removed that sentence: 
>This can mean little or no custom development.

And 4 lines down, I added:
>There can be challenges, like reimplementing JavaScript that the plugin has to remove to produce valid AMP. But the validator helps you to see which templates are ready for AMP, and which might need development.